style.css

```diff
@@ -724,6 +1509,155 @@ body {
     width: 23px;
     height: 30px;
   }
+
+  .second-page {
+    min-height: 687px;
+  }
+
+  .second-page__img {
+    margin-right: 54px;
+  }
+
+  .second__img_purses {
+    width: 84%;
+    padding: 83px 0 0 104px;
+  }
+
+  .second__wrap {
+    margin: 40px 18px 0 0;
+  }
+
+  .second-page__article {
+    width: 282px;
+    padding: 31px 48px 34px 43px;
+  }
+
+  .second-page__title {
+    font-size: 31px;
+    letter-spacing: -.3px;
+    margin-bottom: 5px;
+  }
+
+  .second-page__subtitle {
+    font-size: 12px;
+    letter-spacing: 1px;
+    margin-bottom: 47px;
+  }
+
+  .second-page__description {
+    font: 600 12px/16px 'Montserrat-Regular', sans-serif;
+    margin-bottom: 30px;
+    letter-spacing: -.3px;
+  }
+
+
+  .second-page__color-options {
+    margin-bottom: 38px;
+  }
+
+  .eclipse {
+    width: 24px;
+    height: 24px;
+    margin-right: 10px;
+  }
+
+  .second-page__price {
+    margin-bottom: 43px;
+    font-size: 30px;
+  }
+
+.second__minus {
+    padding: 4px 8px;
+  }
+
+  .second__quantity__input {
+    width: 6px;
+    font-size: 12px;
+    padding: 3px;
+  }
+
+  .second__plus {
+    padding: 4px 10px;
+  }
+
+  .second__add-to-cart {
+    font-size: 10px;
+    padding: 2px 16px;
+    width: 195px;
+  }
+
+  .second__more-items {
+    font-size: 10px;
+    letter-spacing: 0;
+    padding-left: 3px;
+  }
+
+    .third-page {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)